packages/core/src/adapters.ts

```diff
@@ -216,7 +216,7 @@ export interface VerificationToken {
  * :::
  */
 export interface Adapter {
-  createUser?(user: Omit<AdapterUser, "id">): Awaitable<AdapterUser>
+  createUser?(user: Omit<AdapterUser, "id">, info?: { signinInfo?: SigninInfo }): Awaitable<AdapterUser>
```

Comment:
I add it as an optional param so it may be extended in the future with other data. Also shouldn't break exiting adapters.